chain-core/src/lib.rs

```diff
@@ -0,0 +1,12 @@
+#[macro_use]
+extern crate cfg_if;
+
+cfg_if! {
+    if #[cfg(test)] {
+        extern crate quickcheck;
+    } else if #[cfg(feature = "property-test-api")] {
```

Comment:
The feature is named `property-test-apis` in Cargo.toml. I think this name is better, though.

So you think it may be useful to provide quickcheck APIs to non-dev dependencies? This PR in current state does not use the feature to do it, but exports the test module in the `test` configuraition, which I think is not ideal (see below).